assets/react/controllers/NavBarCart.jsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import CartTop from '../../../js/app/cart/CartTop'
+
+export default function NavBarCart(props) {
```

Comment:
- why is this change needed?
- should we add this for all the components we attach to the dom ?

Reply:
It's an experiment of using Symfony\UX\React\ReactBundle to render react components in twig templates instead of adding them in scripts: https://github.com/coopcycle/coopcycle-web/pull/3979/files#diff-540f9bd9a739c50f0821fcd03e7894fcf56f3a8e0a22acbe7423092cfd33d093R77-R81

For now, twig will only pickup components from `assets/react/controllers/` that's why this wrapper is needed. Later on we can set it up to look for components in `/js/app`, but it might have consequences on the code splitting